server/schema/assigntsigkey.sql

```diff
@@ -1,12 +1,12 @@
 CREATE OR REPLACE FUNCTION AssignTSIGKey(
 	domain varchar,
```

Comment:
@nemanja-zivkovic-atomia It is possible to assign the tsig key that doesn't exist from atomiadns. This is not a valid situation from HCP. I am not sure if this should be fixed?

Reply:
It is possible, only trough AtomiaDNS. It is not possible trough HCP.

@sasa-mladenovic-troxo I don't remember if we discussed this, but should we prevent this situation? We can do so by adding a foreign key in domainmetadata on tsigkey table.

Reply:
This seems to be allowed on PowerDNS as well. I'm fine to leave it like this for now.